### ISSUE TITLE
`publish doc` copies the wrong directory

### ISSUE THREAD

**Issue by Drup** [samoht/dune-release#16]
There are 3 subdirectories under `doc/` in the build artifacts: `_html`, `_odoc`, `_mld`. dune-release currently copies all three, instead of copying just the content of `_html`.

Additionally, it would be nice to ask confirmation where the doc should go before actually copying it and pushing.

**Comment by mmottl** [comment]
I just ran into this with `topkg`: https://github.com/dbuenzli/topkg/issues/129

@Drup 's solution of just copying the contents of `_html` would work fine for me.

**Comment by Chris00** [comment]
Same [for me](https://github.com/dbuenzli/topkg/issues/132).

**Comment by samoht** [comment]
Which version of `odoc` are you using?

With `odoc.1.2.0` and the master branch of `dune-release`, I got:

```
$ dune-release publish doc --dry-run
[...]
$ ls _build/dune-release-0.1.0-27-ga27e4a3/_build/default/_doc
dune-release	index.html	odoc.css
```

**Comment by Drup** [comment]
Pretty sure I was using the latest release *at the time* for odoc, dune-release and jbuilder. I'll see if I still have the issue next time I release something. :)

**Comment by Chris00** [comment]
I pinned `dune-release`.
```
$ odoc --version
1.2.0
$ dune-release publish doc --dry-run
[...]
$ ls _build/lpd-1.2.2-1-g8469692/_build/default/CHANGES.md  lpd.cmxa     lpd.ml     lpd.socket.dune   socket.a      socket.ml
LICENSE.md  lpd.cmxs*    lpd.ml.d   lpd.version.sexp  socket.cma    socket.ml.d
lpd.a       lpd.dune     lpd.mli    META.lpd          socket.cmxa   socket.mli
lpd.cma     lpd.install  lpd.mli.d  README.md         socket.cmxs*  socket.mli.d
```
so no `_doc` directory.  However, after `jbuilder build @doc`, I get
```
$ ls _build/default/_doc/
_html/  _mlds/  _odoc/
```
and  `dune-release publish` pushes the [same directory structure](https://github.com/Chris00/lpd/tree/gh-pages).

Note that, `dune-release publish` finishes with
```
dune-release: [ERROR] Could not find release id in response:
                      HTTP/1.1 404 Not Found
                      Server: GitHub.com
                      Date: Sat, 09 Jun 2018 18:40:34 GMT
                      Content-Type: application/json;
                      charset=utf-8
                      Content-Length: 120
                      Status: 404 Not Found
                      X-RateLimit-Limit: 5000
                      X-RateLimit-Remaining: 4945
                      X-RateLimit-Reset: 1528571402
                      X-OAuth-Scopes: repo
                      X-Accepted-OAuth-Scopes: repo
                      X-GitHub-Media-Type: github.v3;
                      format=json
                      Access-Control-Expose-Headers: ETag, Link, Retry-After,
                      X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining,
                      X-RateLimit-Reset, X-OAuth-Scopes,
                      X-Accepted-OAuth-Scopes,
                      X-Poll-Interval
                      Access-Control-Allow-Origin:
                      *
                      Strict-Transport-Security: max-age=31536000;
                      includeSubdomains; preload
                      X-Frame-Options: deny
                      X-Content-Type-Options: nosniff
                      X-XSS-Protection: 1; mode=block
                      Referrer-Policy: origin-when-cross-origin,
                      strict-origin-when-cross-origin
                      Content-Security-Policy: default-src
                      'none'
                      X-Runtime-rack: 0.028666
                      X-GitHub-Request-Id:
                      A740:57C5:3BDE7BF:7B46D48:5B1C1F22
                      
                      {
                        "message": "Not Found",
                        "documentation_url":
                      "https://developer.github.com/v3/repos/releases/#create-a-release"
                      }
                      .
```

**Comment by samoht** [comment]
@Chris00 which version of `odoc` do you have?

**Comment by samoht** [comment]
@Chris00 also does `dune-release publish distrib` seems to make sensible calls? (e.g. to the right repository)

**Comment by Chris00** [comment]
> @Chris00 which version of odoc do you have?

1.2.0

> does `dune-release publish distrib` seems to make sensible calls? (e.g. to the right repository)

It seems so:
```
$ dune-release publish distrib --dry-run
=> must exists _build/lpd-1.2.2.tbz
-: exec: git --git-dir .git rev-parse --verify 1.2.2
-: exec: git --git-dir .git push --force git@github.com:Chris00/lpd.git 1.2.2
=> read /home/trch/.dune/github.token
-: exec:
     curl -s -S -K - -D - --data   { "tag_name" : "1.2.2", "body" : "CHANGES:\n\n- Use jbuilder to compile.\n- Fix safe-string issues so it works with OCaml ≥ 4.06\n" }   https://api.github.com/repos/${user}/${repo}/releases
=> read /home/trch/.dune/github.token
-: exec:
     curl -s -S -K - -H Content-Type:application/x-tar --data-binary   @_build/lpd-1.2.2.tbz   https://uploads.github.com/repos/${user}/${repo}/releases/0/assets?name=lpd-1.2.2.tbz
```

**Comment by Chris00** [comment]
I used `dune-release publish` on another repository and it worked fine.  I'll have to investigate what it the problem with [lpd](https://github.com/Chris00/lpd).

**Comment by samoht** [comment]
A tentative fix is in #31. Not sure why `odoc` sometimes generates an `_html` subdir and sometimes not..
